pose_format/utils/holistic.py

```diff
@@ -52,7 +53,7 @@ def process_holistic(frames: list, fps: float, w: int, h: int, kinect=None, prog
         results = holistic.process(frame)
 
         body_data, body_confidence = body_points(results.pose_landmarks, w, h, 33)
-        face_data, face_confidence = component_points(results.face_landmarks, w, h, face_num)
+        face_data, face_confidence = component_points(results.face_landmarks, w, h, 468 + additional_face_points)
```

Comment:
use `FACE_POINTS_NUM(additional_face_points)` to avoid duplication of the constant `468`